CMakeLists.txt

```diff
@@ -101,10 +101,14 @@ else ()
         elseif(NOT USE_OPENSSL AND IN_SOURCE_BUILD)
             if (TARGET crypto)
                 message(STATUS "Using libcrypto from AWS-LC")
+                set(PLATFORM_LIBS crypto)
+            elseif(aws-lc-PREBUILT)
```

Comment:
I would use a different symbol name here.  Mixing the casing and using `-` feels weird even if cmake is ok with it.